backend/src/core/config.ts

```diff
@@ -154,6 +154,12 @@ interface ConfigSchema {
     maxAttachmentNum: number
     maxCumulativeAttachmentsSize: number
   }
+
+  phonebook: {
```

Comment:
kill switch not required here ?

Reply:
Intention is that we can control backend's killswitch by hiding UI of frontend instead. (Returning empty list from agency list endpoint will disable the ui as well)